guides/changing-metadata-store.md

```diff
@@ -0,0 +1,14 @@
+# Changing Metadata Store
+
+Aqueduct uses a storage layer for tracking workflow, operator, and artifact metadata. The default storage layer is the local filesystem. Aqueduct currently also supports using the following services for its metadata store:
```

Comment:
Let's add a little bit here about why the user might want to use a non-FS metadata store (and also when it's required).